content/updates/2016-12-28-release-notes.md

```diff
@@ -0,0 +1,18 @@
++++
+date = "2016-12-28"
+title = "Platform Release Notes"
++++
+
+Curious what’s new that you might find helpful as a cloud.gov application developer? Here are highlights from our platform updates over the past two weeks.
+<!--more-->
+
+### Added
+- [S3 service keys]({{< relref "docs/services/s3.md#get-s3-bucket-credentials" >}}) for direct access to your cloud.gov S3 buckets can now be created.
```

Comment:
Yay! Suggest "You can now create [S3 service keys]({{< relref "docs/services/s3.md#get-s3-bucket-credentials" >}}) to get direct access to your cloud.gov S3 buckets."